receiver/k8sobjectsreceiver/config.go

```diff
@@ -59,8 +59,9 @@ type K8sObjectsConfig struct {
 type Config struct {
 	k8sconfig.APIConfig `mapstructure:",squash"`
 
-	Objects   []*K8sObjectsConfig `mapstructure:"objects"`
-	ErrorMode ErrorMode           `mapstructure:"error_mode"`
+	Objects             []*K8sObjectsConfig `mapstructure:"objects"`
+	ErrorMode           ErrorMode           `mapstructure:"error_mode"`
+	IncludeInitialState *bool               `mapstructure:"include_initial_state"`
```

Comment:
Why pointer?

Reply:
Why do we need unset?

Reply:
I wanted to handle when it's unset vs. set to false, but it's not needed, so just refactored it to bool.